homeassistant/components/lg_thinq/strings.json

```diff
@@ -23,6 +23,23 @@
       "operation_power": {
         "name": "Power"
       }
+    },
+    "binary_sensor": {
+      "eco_friendly_mode": {
+        "name": "Eco Friendly"
+      },
+      "power_save_enabled": {
+        "name": "Power Saving Mode"
+      },
+      "remote_control_enabled": {
+        "name": "{location}Remote Start"
```

Comment:
Please use Sentence case for casing

Reply:
{location} Removed. 
As you mentioned at https://github.com/home-assistant/core/pull/125054#discussion_r1740516115, placeholder is not passed.

Reply:
Modified.